components/base/__tests__/TabBarPanel.test.tsx

```diff
@@ -0,0 +1,54 @@
+import renderer from "react-test-renderer";
+import TabBarPanel from "../TabBarPanel";
+import phpGray from "../../assets/images/content-icons/php-gray.svg";
+import pythonGray from "../../assets/images/content-icons/python-gray.svg";
+import rubyColor from "../../assets/images/content-icons/ruby-color.svg";
+import rubyGray from "../../assets/images/content-icons/ruby-gray.svg";
+import cSharpGray from "../../assets/images/content-icons/c-sharp-gray.svg";
+import kotlinGray from "../../assets/images/content-icons/kotlin-gray.svg";
+import nodeGray from "../../assets/images/content-icons/node-gray.svg";
+
+const content = [
+  {
+    label: "node",
+    icon: nodeGray,
+    iconActive: nodeGray,
+    content: "Here's some content related to Node.",
+  },
+  {
+    label: "ruby",
+    icon: rubyGray,
+    iconActive: rubyColor,
```

Comment:
This is an example of passing both the gray and the colored icons to the component. Since we don't have the colored icons for the other languages yet, I ended up passing the same gray icons twice for now.